src/redux/api/backendApi.js

```diff
@@ -0,0 +1,13 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { API_BACKEND } from '../../helpers/Api';
+
+export const backendApi = createApi({
```

Comment:
Can we cache calls here?

Reply:
All calls are cached by RTK automatically.